File: selenium_sample.js
test.describe( 'Test Suite' , function(){
    WebDriver driver = new ChromeDriver();

    test.it( 'Test Case' , function(){
        driver.get("https://dev.sdvi.com");
        driver.findElement(webdriver.By.id("resource1TabId")).click();
        var descriptionElement = driver.findElement(By.id("resourceDescription"));
        var text = descriptionElement.getText();
        expect(text).equals("This is the description of item 1");

        driver.findElement(By.id("resource2TabId")).click();
        text = descriptionElement.getText();
        expect(text).equals("This is the description of item 2");

        driver.sleep();
    });

});

File: selenium_sample.py
def test_step(self):
    driver = self.driver
    driver.get("https://dev.sdvi.com")

    driver.find_element_by_id('resource1TabId').click()
    descriptionElement = driver.find_element_by_id('resourceDescriptionId')
    descriptionText = descriptionElement.text
    self.assertTrue("Description of resource 1" in descriptionText)

    driver.find_element_by_id('resource2TabId').click()
    descriptionText = descriptionElement.text
    self.assertTrue("Description of resource 2" in descriptionText)